build.gradle

```diff
@@ -46,13 +47,19 @@ allprojects {
     group = 'com.netflix.graphql.dgs.codegen'
 
     dependencies {
+        // Gradle Platforms applied to all modudles
+        api platform("com.netflix.graphql.dgs:graphql-dgs-platform-dependencies:${Versions.DGS_VERSION}")
+
+        implementation platform("com.netflix.graphql.dgs:graphql-dgs-platform-dependencies:${Versions.DGS_VERSION}")
```

Comment:
it doesn't hurt to have both `api` and `implementation`, but the latter is redundant given the former

Reply:
Fixed, just kept one and removed the `Versions.DGS_VERSION` variable. This will help the depbot identify when the BOM should be updated.